lp-lsp/lp_lsp.ml

```diff
@@ -29,8 +29,8 @@ module List = struct
 end
 
 module Option = struct
```

Comment:
Why not reusing Extra.Option instead?

Reply:
This module is not used anymore. Remove it?

Reply:
We can remove it if you want, but it tends to re-appear pretty often.